src/executables/util_service.c

```diff
@@ -4255,6 +4258,14 @@ process_heartbeat (int command_type, int argc, const char **argv)
 #if !defined(WINDOWS)
   HA_CONF ha_conf;
 
+  if (ha_mode_in_common == HA_MODE_OFF)
```

Comment:
How about using util_get_ha_mode_for_sa_utils () instead of the global variable 'ha_mode_in_common'?
It seems to be used to check this case in other codes like 'process_server()'.
`if ( util_get_ha_mode_for_sa_utils () == HA_MODE_OFF) 
{
...
`

Reply:
@wrlawodms In my opinion, there is a design level problem with the logic that handles the configuration. the return value of the util_get_ha_mode_for_sa_utils() function at this point becomes HA_MODE_OFF when executing the 'cubrid service start' command after configuring cubrid.conf as follows

>[common]
ha_mode=on
...
[@db1]
ha_mode=on
...
[@db2]
ha_mode=off

this is because properties that can be set for each section are handled by using one global variable.